src/amo/components/Link.js

```diff
@@ -18,7 +19,12 @@ export class LinkBase extends React.Component {
   }
 
   render() {
-    const { base, children, to } = this.props;
+    const { base, children, href, to } = this.props;
```

Comment:
I think it would be helpful to throw an error if *both* `to` and `href` are defined because that would probably be a mistake. I can see making that mistake if I was changing a lot of anchor tags to `Link`s maybe.